mysql/metadata.csv

```diff
@@ -1,5 +1,7 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
 mysql.info.schema.size,gauge,,mebibyte,,"Size of schemas in MiB",0,mysql,mysql schema size
+mysql.info.table.index_size,gauge,,mebibyte,,"Size of tables index in MiB",0,mysql,mysql index table size
```

Comment:
For postgres we send table size metrics in terms of just bytes instead of MiB. Should we make the unit consistent across dbms types?

Reply:
ah that makes sense. Sounds fine to keep it consistent with the existing metric 👍